minutes/21-03-05

```diff
@@ -0,0 +1,28 @@
+MIABIS working group update 05.03.21
+
+Attendees: Andrzej Strug, Aneas Hodselsmans, Blazej Marciniak, Cäcilia Engels, Christoph Döllinger, EJ. Vinke, Gabi Anton (GA), Glerean Enrico, Hans Demski, Joanna Vella, Jörg Geiger, Kristjan Metsalu, Michael Neumann (MN), Noemi Deppenwiese, Lauri Pakonnen, Kaisa Silander, Tomas Snackerstrom, Zdenka Dudová
+
+# Introduction to the "star model" (MN)
+ - star model is no typical relational data model for normalisation generally used to reduce redundancy
+ - the star model is customary in data ware houses, where it is optimal for querying data: simple, fast, redundant
+ - the model emploies with dimension-tables and facts-tables working towards aggregation 
```

Comment:
the model is designed so that one facts-table carries the aggregation measures whereas several dimension tables carry the filter criteria